### PR TITLE
docs: Bring e/sql docs up to date after recent module reorg.

### PR DESCRIPTION
**Pull request by dpnova**
## Type

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update

## Description

Simply updating the `e/sql` docs to reflect the recent move of namespaces.

I realised I used some variations of imports. I thought it could be useful for people to know. Happy to stick to a consistent blessed import style if required though, just let me know.

## Related
This is the follow on from #3035

- Related Issue #3035

### CODE REVIEW

File: packages/sql/README.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ A SQL toolkit for Effect.
 import { Config, Effect, Struct, pipe } from "effect"
 import * as Sql from "@effect/sql"
 import * as Pg from "@effect/sql-pg"
+import { SqlClient } from "@effect/sql/SqlClient"
```
Comment (tim-smart) [inline packages/sql/README.md:11]:
```suggestion
import { SqlClient } from "@effect/sql"
import { PgClient } from "@effect/sql-pg"
```

This is the recommended import style.

**Review by tim-smart — CHANGES_REQUESTED** [review]
Thanks! Could you update it to use `import { ... }` imports?

File: packages/sql/README.md (hunk 1 above)
Comment (dpnova) [reply packages/sql/README.md:11]: Yep, can do. Thanks for the quick feedback.

Comment (dpnova) [reply packages/sql/README.md]: Shall I get rid of the odd inline comment here? https://github.com/Effect-TS/effect/blob/main/packages/sql/README.md?plain=1#L106

**Comment by dpnova** [comment]
With the things that have been moved - are you comfortable with the way they look considering they don't match what you would import anymore? https://github.com/Effect-TS/effect/blob/main/packages/sql/README.md?plain=1#L83

**Comment by tim-smart** [comment]
> With the things that have been moved - are you comfortable with the way they look considering they don't match what you would import anymore? https://github.com/Effect-TS/effect/blob/main/packages/sql/README.md?plain=1#L83

The resolver APIs would now be `SqlResolver.findById` etc

File: packages/sql/README.md [hunk 2]
```diff
@@ -7,23 +7,22 @@ A SQL toolkit for Effect.
 ```ts
 import { Config, Effect, Struct, pipe } from "effect"
 import * as Sql from "@effect/sql"
```
Comment (tim-smart) [inline packages/sql/README.md:9]: You can remove this import now.

**Review by tim-smart — CHANGES_REQUESTED** [review]

File: packages/sql/README.md (hunk 2 above)
Comment (dpnova) [reply packages/sql/README.md]: woops - code in docs is annoying, I need my damn linter :D

don't want to bother you with reviews, but I think that's it...

**Comment by tim-smart** [comment]
```diff
diff --git a/packages/sql/README.md b/packages/sql/README.md
index 3c9b1be73..7ff7f42c3 100644
--- a/packages/sql/README.md
+++ b/packages/sql/README.md
@@ -37,9 +37,9 @@ For example, to create the client Layer, instead of:
 
 ```ts
 import { Config } from "effect"
-import * as Pg from "@sqlfx/pg"
+import { PgClient } from "@sqlfx/pg"
 
-const SqlLive = Pg.makeLayer({
+const SqlLive = PgClient.layer({
   database: Config.succeed("effect_pg_dev")
 })
 ```
@@ -198,7 +198,7 @@ export const make = (limit: number) =>
 ### Unsafe interpolation
 
 ```ts
-import * as Effect from "effect/Effect"
+import { Effect } from "effect"
 import { SqlClient } from "@effect/sql"
 
 type OrderBy = "id" | "created_at" | "updated_at"
```

**Comment by dpnova** [comment]
> ```diff
> diff --git a/packages/sql/README.md b/packages/sql/README.md
> index 3c9b1be73..7ff7f42c3 100644
> --- a/packages/sql/README.md
> +++ b/packages/sql/README.md
> @@ -37,9 +37,9 @@ For example, to create the client Layer, instead of:
> ```

Blargh hate it when colleagues miss more instances of what I've pointed out. I appreciate your patience mate.

I also haven't used sqlfx - wasn't sure how in lockstep it was. 

Hope that's it... and thanks for the help.

**Review by tim-smart — APPROVED** [review]
Thanks!

**Pull request merged**